openstack/python/tar/bin/openstack-describe-instances

```diff
@@ -32,7 +32,8 @@ class OpenStackDescribeInstances(DescribeInstancesCommand, OpenStackCommand):
                  2: 'Terminated',
                  3: 'Pending',
                  4: 'Unknown',
-                 5: 'Stopped'}
+                 5: 'Stopped',
+                 7: 'Error'}
```

Comment:
@konstan: Can you complete the `STATE_MAP` with the following [libcloud.compute.base.NodeState](https://libcloud.readthedocs.org/en/latest/apidocs/libcloud.compute.html?highlight=nodestate#libcloud.compute.base.NodeState) ?